app/Http/Controllers/ArtworkController.php

```diff
@@ -42,7 +42,8 @@ public function store(Request $request)
         'end_date' => $validatedData['end_date'],
         'users_id' => Auth::id(),
         'category_id' => $validatedData['category_id'],
-        'status' => false,
+        'status' => 'pending',
```

Comment:
Can you check if you can remove this. I believe you already set a default status value on database migration config.